sourcegraph/sourcegraph
# Tree: add generic icons

## Description
Part of the [Tree improvements](https://github.com/sourcegraph/sourcegraph/issues/38667), it [adds icons](https://github.com/sourcegraph/sourcegraph/issues/38664). Ideally, we would have distinct icons per file type but for now it's either of:
- open/closed folder
- generic document icon



https://user-images.githubusercontent.com/2196347/181752912-a5fb4a1a-dd7b-43c7-983a-7fd0d5b510ca.mov



## Test plan
1. Check that tree renders directory and file icons
2. Switch to dark mode and check the colors are correct


## App preview:

- [Web](https://sg-web-og-tree-icons.onrender.com)
- [Storybook](https://5f0f381c0e50750022dc6bf7-cmvxhxtlmy.chromatic.com)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.
